### DISCUSSION TITLE
Deprecate function values in utils since it became part of JS

### DISCUSSION THREAD

**Discussion by cirossmonteiro** [josdejong/mathjs#3194] [Ideas]
[This function](https://github.com/josdejong/mathjs/blob/develop/src/utils/object.js#L393) could be removed and its calls updated to use [current](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values) standard of Javascript.

**Comment by josdejong** [comment]
Good point! Fixed via 5c12c07e2c5a97a234a51a9f6e4b0dbc11afe9c4

**Reply by cirossmonteiro** [reply]
@josdejong  Nice! Another ocurrence [here](https://github.com/josdejong/mathjs/blob/develop/tools/entryGenerator.js#L536).

**Reply by josdejong** [reply]
Ah, yes. Can you find even more occurences?

**Reply by cirossmonteiro** [reply]
Nope, I think that's the last one.

**Reply by cirossmonteiro** [reply]
BUT... BUT... I guess the same replacements could be applied on array.[contains](https://github.com/josdejong/mathjs/blob/a9fbb3560c0a4aa11f79c8584f04e7c7ad044155/src/utils/array.js#L659)
since JS has array.[includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

**Reply by josdejong** [reply]
Ok addressed via e37bacda366872e6dfd16a56f7a8ad344f1b1bc8 and d97f338fccd5fa6bdf591d34946a8794997fcb1b.

I've also replaced occurrences of `.indexOf(...) === -1` and `.indexOf(...) !== -1` with `.contains(...)`.
